### PR TITLE
budgie-desktop: update to 10.7.

### PR DESCRIPTION
**Pull request by BlindRepublic**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by paper42** [comment]
please rebase this PR on top of master to get rid of the merge commit

**Comment by TrueTechie** [comment]
> please rebase this PR on top of master to get rid of the merge commit

Fixed!

File: srcpkgs/budgie-desktop/template [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-revision=2
+version=10.7
+revision=1
 build_style=meson
 build_helper=gir
 configure_args="-Dwith-gtk-doc=false"
 hostmakedepends="pkg-config intltool vala glib-devel gobject-introspection sassc
  budgie-screensaver"
-makedepends="alsa-lib-devel libnotify-devel accountsservice-devel libpeas-devel
+makedepends="alsa-lib-devel libcanberra-devel libgee-devel libnotify-devel accountsservice-devel libpeas-devel
```
Comment (paper42) [inline srcpkgs/budgie-desktop/template:10]: wrap at 80 columns

**Comment by BlindRepublic** [comment]
Apologies for the extra commits, mixed up whitespace.

**Comment by JoshStrobl** [comment]
Hey @BlindRepublic, I noticed that this was just closed without an update to 10.7 or 10.7.1, and missing some updates for other Budgie bits. Any chance you could provide some clarification on if the plan is to get this updated in Void? I'd like to get it added to the new [Buddies of Budgie "Getting Budgie" page](https://docs.buddiesofbudgie.org/user/getting-budgie on our documentation center, but want to make sure it has a maintainer behind it first before people risk installing an old release.

I don't use Void myself but do let me know if there is something I can do to help you.

Thanks!

**Comment by BlindRepublic** [comment]
> Hey @BlindRepublic, I noticed that this was just closed without an update to 10.7 or 10.7.1, and missing some updates for other Budgie bits. Any chance you could provide some clarification on if the plan is to get this updated in Void? I'd like to get it added to the new [Buddies of Budgie "Getting Budgie" page](https://docs.buddiesofbudgie.org/user/getting-budgie on our documentation center, but want to make sure it has a maintainer behind it first before people risk installing an old release.
> 
> I don't use Void myself but do let me know if there is something I can do to help you.
> 
> Thanks!

Hi @JoshStrobl,

Budgie doesn't have a maintainer on Void. I'm just a regular user who likes the desktop. I don't have much experience with Github and I kind of messed up my repo. I fixed the issue afterwards, but this has stayed under review for three weeks...

Either way I appreciate you reaching out. I'll re-open it later today when I have time and try to fix it, but that's about as much as I can do.

**Comment by paper42** [comment]
We are definitely interested in merging this update when this PR doesn't contain unrelated changes and isn't closed.

**Comment by BlindRepublic** [comment]
> We are definitely interested in merging this update when this PR doesn't contain unrelated changes and isn't closed.

Okay... I had to force push and thus could not reopen this request. I won't do so again, but I recreated one at #42621.

Apologies for all of my mistakes.

**Pull request closed without merging**